code/weapon/weapons.cpp

```diff
@@ -627,15 +627,26 @@ void parse_wi_flags(weapon_info *weaponp, flagset<Weapon::Info_Flags> preset_wi_
     if (!optional_string("$Flags:"))
         return;
 
+	// have to do this slightly out of order in order to handle spawn properly
```

Comment:
Looks like this block has different indentation, which CI is complaining about

Reply:
WOW, this is scuffed. I knew the whitespace got rendered weirdly on github, but man... the whole function is a mess of tab vs space mismatch, who on EARTH does an *8 spaces tab*????